integrations/azure-devops/.port/resources/blueprints.json

```diff
@@ -52,6 +54,63 @@
       },
       "required": []
     },
+    "mirrorProperties": {
+      "defaultTeam": {
+        "title": "Default Team",
+        "path": "project.defaultTeam"
+      }
+    },
+    "calculationProperties": {},
+    "aggregationProperties": {},
+    "relations": {
+      "project": {
+        "title": "Project",
+        "target": "project",
+        "required": true,
+        "many": false
+      }
+    }
+  },
+  {
+    "identifier": "project",
+    "title": "Project",
+    "icon": "AzureDevops",
+    "schema": {
+      "properties": {
+        "state": {
+          "title": "State",
+          "type": "string",
+          "icon": "AzureDevops",
+          "description": "The current lifecycle state of the project."
+        },
+        "revision": {
+          "title": "Revision",
+          "type": "string",
+          "icon": "AzureDevops",
+          "description": "The revision number, indicating how many times the project configuration has been updated."
+        },
+        "visibility": {
+          "title": "Visibility",
+          "type": "string",
```

Comment:
same here order should be changed, you defined the repo with mirror property to project, before the project got even created